blazor/dialog/events.md

```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Events in Blazor Dialog Component
 
-This section explains the list of events of the Dialog component which will be triggered for appropriate Dialog actions.
+This section explains the list of events of the [Blazor Dialog](https://www.syncfusion.com/blazor-components/blazor-modal-dialog) which will be triggered for appropriate Dialog actions.
```

Comment:
Use the below:

This section explains the list of events of the [Blazor Dialog](https://www.syncfusion.com/blazor-components/blazor-modal-dialog) component which will be triggered for appropriate Dialog actions.